webclient/components/ptinfo-single/1time-Mrow-mField/diagnosis/edit-layer/edit-design-1.vue

```diff
@@ -82,24 +82,33 @@ export default {
         .where('ROW_END', 2147483648000)
         .get()
     },
-    fnSelectionToggle(rowData) {
+    async fnSelectionToggle(rowData) {
```

Comment:
This logic is not correct. 
Logic should be:
1. On load edit layer, insert a empty row in client side table
2. update inserted row on the time of typing in the form

Follow reminder component

Reply:
2. Don't have any text type of field in the diagnosis form